molecule/disable_stub_resolver/tests/test_disable_stub_resolver.py

```diff
@@ -0,0 +1,29 @@
+"""Module containing the tests for the default scenario."""
+
+# Standard Python Libraries
+import os
+
+# Third-Party Libraries
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ["MOLECULE_INVENTORY_FILE"]
+).get_hosts("all")
+
+
+def test_port_53_listener(host):
+    """Verify that nothing is listening on port 53."""
+    # Have netstat output all TCP and UDP IPv4 listeners.
+    netstat_cmd = "netstat --protocol inet --listening --tcp --udp --numeric"
```

Comment:
If we want the options listed in alphabetical order:
```suggestion
    netstat_cmd = "netstat --protocol inet --listening --numeric --tcp --udp "
```